docs/DeveloperGuide.md

```diff
@@ -566,15 +566,20 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### Use cases
 
 (For all use cases below, the **System** is the `ParentPal` and the **Actor** is the `user`, unless specified otherwise)
+For our product, we have two different entities `Contact` and `Appointment`. Across the two entities,
+some implementations are rather similar.
+Therefore, for those implementations, we will use `ENTITY` to generalise the different entities and then provide use
```

Comment:
Perhaps instead of monospace, using italics would be better to differentiate it from actual code-related terms?